### PR TITLE
Pkgs

### PR DESCRIPTION
**Pull request by scopatz**
Here are a few updates to how packages are loaded (now lazy) and how the Package object is internally structured. This will help the REST API and the consumers

### CODE REVIEW

**Comment by scopatz** [comment]
CC @dougalsutherland @justcalamari @mariusvniekerk @CJ-Wright

File: news/pkgs.rst [hunk 1]
```diff
@@ -0,0 +1,26 @@
+**Added:**
+
+* New ``ChannelGraph`` model for representing the graph for a given channel.
+* Database ``DB`` objcet has new ``channel_graphs`` attribute, which is a
```
Comment (CJ-Wright) [inline news/pkgs.rst]: object

Comment (scopatz) [reply news/pkgs.rst]: thanks, fixed

**Comment by scopatz** [comment]
pinging this again! It is blocking other progress on REST stuff

**Comment by scopatz** [comment]
Is there any reason to wait for @mariusvniekerk here that I am not seeing? Not that I mind, per se, but it seems like having one reviewer would be enough for most things.

**Comment by scopatz** [comment]
Thanks!

**Pull request merged**